department-of-veterans-affairs/abd-vro
# Feature - Update Contention Classification Documentation

## What was the problem?

Previously, there was a place where some FastAPI python code lived, but it did not follow the same structure as VRO guidelines. It also didn't resemble what is actually needed for API spec.

Associated tickets or Slack threads:

- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/84

## How does this fix it?[^1]


## How to test this PR
- Run vro-app-1 and navigate to http://localhost:8080/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#/
- Notice description under CC domain and link to building Contention Classification Swagger docs
- Run the contention classification code and navigate to http://localhost:18000/docs
- Notice `/classifier` endpoint w/ input and outputs


## Screenshots
From http://localhost:8080/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#/
<img width="980" alt="Screenshot 2023-05-08 at 9 36 37 AM" src="https://user-images.githubusercontent.com/17188013/236881346-610b9821-eb60-43ae-b231-ec43af813a8c.png">

From http://localhost:18000/docs
<img width="1311" alt="Screenshot 2023-05-08 at 9 44 06 AM" src="https://user-images.githubusercontent.com/17188013/236881682-4e025ff6-9ac4-4bae-84f6-36b82ecdf993.png">
<img width="834" alt="Screenshot 2023-05-08 at 9 44 10 AM" src="https://user-images.githubusercontent.com/17188013/236881696-cef25e92-1edb-4f89-83f9-239ca8d6c64e.png">


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
